README.md

```diff
@@ -51,6 +54,22 @@ This repository contains build files for the FreeRTOS Windows port - able to be
 
 In both cases, the generated executable can be found at `build/Cortex-M3_MPS2_QEMU_GCC/output/RTOSDemo.elf`.
 
+**Running the makefile build in QEMU**
+
+Use the QEMU command line contained in the [start_qemu.bat](https://github.com/FreeRTOS/coreMQTT-Agent-Demos/blob/main/build/Cortex-M3_MPS2_QEMU_GCC/start_qemu.bat) batch file to run the generated 'RTOSDemo.elf' file directly.  Alternatively use the command line contained in the [start_qemu_and_wait.bat](https://github.com/FreeRTOS/coreMQTT-Agent-Demos/blob/main/build/Cortex-M3_MPS2_QEMU_GCC/start_qemu_and_wait.bat) batch file to start QEMU and wait for a GBD connection on port 1234.  The Eclipse project contains a suitable debug configuration.
+
+Note these QEMU command lines assume there is a network tap driver called "TAP0".
+
+**Obtaing network access from QEMU on windows**
+
+1. Install the [OpenVPN TAP driver for Windows](https://openvpn.net/).
+
+1. Name the installed TAP interface "TAP0".  This can be done by selecting "Change adaptor settings" from the Windows' Network Properties window (Control Panel\Network and Internet\Network Connections) to view all network adaptors, selecting the installed TAP adapter, then pressing F2 to edit the adaptor's name.
+
+1. From the same window, configure a real wired network interface to have a static IP address that is compatible with whichever IP address would otherwise be allocated by a DHCP server on your local network.
```

Comment:
This is a little confusing. Does this mean get an address from the DHCP server and assign it as a static address, choose an available IP address from the DHCP server's range, or choose an IP address that does **not** conflict with the DHCP server's range? It sounds like it could be the second one, but I'm not sure
```suggestion
1. From the same window, configure a real wired network interface to have a static IP address that is within the range of IP addresses allocated by your local DHCP server.
```